### ISSUE TITLE
any new update?

### ISSUE THREAD

**Issue by Zyriix** [lucidrains/lion-pytorch#14]
any new update for this optimizer?

**Comment by GeneralAwareness** [comment]
Is this really an optimizer?  I am being serious because 30-300% slower (depending on the card) and it devours your vram with loss rates from hell, I just can't consider this an optimizer (nor should anyone).

Is this being implemented correctly?

**Comment by xiangning-chen** [comment]
Hi, thanks for trying out. What's your task and how did you use it?
Lion should consumes less memory than Adam as it only tracks momentum, it also has faster runtime (steps/sec).

**Comment by GeneralAwareness** [comment]
> Hi, thanks for trying out. What's your task and how did you use it? Lion should consumes less memory than Adam as it only tracks momentum, it also has faster runtime (steps/sec).

Nope, every single person who has tried with StableDiffusion is griping about the memory usage and poor performance.  If you say different then I must surmize someone (one of the devs) has implemented this wrong and is giving this a very bad name across the boards/forums.

**Comment by DamonianoStudios** [comment]
I have noticed that using a batch size of 12 with AdamW8bit uses about 11gb of VRAM with gradient checkpointing on, while the same batch size with gradient checkpointing uses a total of 17gb of VRAM with the Lion Optimizer on my RTX 3090.

**Comment by xiangning-chen** [comment]
Hi, are you also using 8 bit training with Lion?

**Comment by GeneralAwareness** [comment]
No, we can't use both it is either or, at least for me it is (it is a drop-down and you select which one to use).

**Comment by lucidrains** [comment]
@Zyriix inconclusive, come back in another month. it takes time
